tests/app/main/views/test_index.py

```diff
@@ -165,3 +190,30 @@ def test_landing_page_has_supplier_contact_info(client, mocker, sample_service,
         assert page.findAll(text=re.compile(expected_result))
     else:
         assert page.findAll(attrs={'href': expected_result})
+
+
+@pytest.mark.parametrize('json_response, expected_result', [
+    ({"file_exists": "True"}, True),
+    ({"file_exists": "False"}, False),
```

Comment:
can we add a `{'error': 'Invalid decryption key'}` test?

Reply:
This case is now tested in two new tests - `test_landing_page_when_checking_document_raises_an_error_shows_appropriate_error_page` & `test_is_file_available_when_document_download_api_gives_a_decryption_key_error`.